File: Solution/WebSolution/Layered/TFLOBL/Entities/Payroll/Salary.cs
namespace Transflower.TFLPortal.TFLOBL.Entities;

public class Salary
{
    public int Id { get; set; }
    public int EmployeeId { get; set; }
    public DateTime PayDate { get; set; }
    public int MonthlyWorkingDays { get; set; }
    public double Deduction { get; set; }
    public double Tax { get; set; }
    public double PF { get; set; }
    public double Amount { get; set; }
 
}

File: Solution/UISolution/TFLUIPortal/src/app/shared/Entities/Payroll/Salary.ts
export class Salary {
    constructor(
      public Id: number,
      public employeeId: number,
      public payDate: string,
      public monthlyWorkingDays: number,
      public deduction: number,
      public tax: number,
      public pf: number,
      public amount: number
    ) {}
  }